example-streamer-uses/Cargo.toml

```diff
@@ -55,6 +68,6 @@ protobuf = { workspace = true }
 serde = { workspace = true }
 tokio = { workspace = true }
 up-rust = { workspace = true }
-up-transport-zenoh = { version = "0.1.1"  }
-up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.1.0", default-features = false }
-zenoh = { version = "1.0.0-alpha.6", features = ["unstable"]}
+up-transport-zenoh = { version = "0.3.0", optional = true }
+up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.3.0", default-features = false, optional = true }
+zenoh = { version = "1.0.0", optional = true }
```

Comment:
Any particular reason why you do not want to declare these at the workspace level and then add the `optional = true` flag at the package level ...

Reply:
Wrote that up [here](https://github.com/eclipse-uprotocol/up-streamer-rust/pull/54#discussion_r1842180856).